packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts

```diff
@@ -654,7 +649,7 @@ function createHba1cFromObservationVitalsSection(observations: Observation[]): {
   });
 
   const observationsLast5Years = hba1cObservations.filter(observation => {
-    return dayjs(observation.effectiveDateTime).isAfter(dayjs().subtract(5, "year"));
+    return dayjs(observation.effectiveDateTime).isAfter(dayjs().subtract(2, "year"));
   });
```

Comment:
update var name. I'm assuming this is _only_ being used by the single cx this change is being made for?

Reply:
Correct